.github/workflows/push_to_master.yml

```diff
@@ -0,0 +1,26 @@
+name: Python linters and code checking
+
+on: 
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  check_code:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
```

Comment:
Ahah, we actually should use 3.6, you know...![IMG_20220330_121717.jpg](https://user-images.githubusercontent.com/67920266/160797212-f41f9d5b-2661-4682-9860-8b960863cc72.jpg)

Reply:
> It think we should use last 3.10 version of python

Last version is cool, but there are people, who do not follow last news. So, they have, for example 3.6 version, but some new features in 3.10 will not work in 3.6. That is why these people will not upgrade the python on their machines just for one package. I'm sure we should use not laster version than 3.8.

Reply:
It seams strange for me to use older python, because most servers use 3.6. Because project development on older python partially forces users not to update they're python. Also many client machines run 3.8 now, because recent distros support it.